release_files/Dockerfile.redhat

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-ARG BASE_IMAGE=registry.access.redhat.com/ubi8/ubi:8.2
+ARG BASE_IMAGE=registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
Can we set particular numeric version for better control over the used base image?